pkg/services/object_manager/placement/netmap.go

```diff
@@ -45,10 +57,21 @@ func (b *netMapBuilder) BuildPlacement(cnr cid.ID, obj *oid.ID, p *netmapSDK.Pla
 	binCnr := make([]byte, sha256.Size)
 	cnr.Encode(binCnr)
 
-	cn, err := nm.GetContainerNodes(p, binCnr)
+	var cn netmapSDK.ContainerNodes
+	if nm == b.lastNm {
+		raw, ok := b.containerCache.Get(string(binCnr))
+		if ok {
+			cn = raw.(netmapSDK.ContainerNodes)
+			return BuildObjectPlacement(nm, cn, obj)
+		}
+	}
+
+	b.containerCache.Purge()
```

Comment:
is it expected to purge the `containerCache` if netmap is the last one and the cache does not contain the requested container?